src/Morphir/JsonSchema/Backend.elm

```diff
@@ -273,14 +273,23 @@ mapType typ =
                         )
 
         Type.Record _ fields ->
+            let
+                requiredFields : List String
+                requiredFields =
+                    fields
+                        |> List.map
+                            (\field ->
+                                field.name |> Name.toCamelCase
+                            )
```

Comment:
We don't want all the fields to be required. Only the ones that are not `Maybe`.